airbrus.mt

```diff
@@ -188,7 +244,7 @@ def main(argv, => Timer,
                 def brusHelp := makeAirbrusHelp(fn s {client.say(channel, s)})
                 def instanceEnv := ["&&help" => &&brusHelp]
                 def userEnv := userEnvironments.fetch(user.getNick(),
-                                                      fn {baseEnv | instanceEnv})
+                                                      fn { return baseEnv | instanceEnv | ["giftExchange" => make_giftExchange_interface(user.getNick())] })
```

Comment:
why add `return` in the `fn`? it's implicit.

Reply:
This is a bug; `return` is not legal inside `fn` on its own, and the code will end up calling the (stale) `__return` ejector from `main`'s frame.